nextcord/file.py

```diff
@@ -13,6 +13,12 @@ class File:
     r"""A parameter object used for :meth:`abc.Messageable.send`
     for sending file objects.
 
+    .. versionadded:: 2.4
+
+        You can now use nextcord.File as a context manager. This will
+        automatically call `close()` when the context manager exits scope.
```

Comment:
This will make a link in the docs. We use Sphinx to generate docs right now, and also ReStructuredText (RST). Here's the link to that "role"'s documentation:
<https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#role-py-meth>
![image](https://user-images.githubusercontent.com/34910574/220777854-ace9c1d8-6db6-4f46-94db-64ad278563ae.png)

```suggestion
        automatically call :meth:`close` when the context manager exits scope.
```